IdentityCore/src/util/NSString+MSIDTelemetryExtensions.m

```diff
@@ -73,4 +73,54 @@ - (NSDictionary *)msidParsedClientTelemetry
     return telemetryDict;
 }
 
++ (NSString *)msidAddPlatformSequenceParamWithName:(NSString *)name version:(NSString *)version toSequence:(NSString *)sequence
+{
+    NSMutableArray *sequenceItems;
+    static const int kSequenceItemsCount = 6;
+    static NSString *const kDelimeter = @",";
+    
+    if ([NSString msidIsStringNilOrBlank:sequence])
+    {
+        sequenceItems = [NSMutableArray new];
+        [sequenceItems addObject:@""]; //Src-SKU
+        [sequenceItems addObject:@""]; //Src-SKU-Ver
+        [sequenceItems addObject:@""]; //Msal-Runtime-Ver
+        [sequenceItems addObject:@""]; //Browser-Ext-Sku
+        [sequenceItems addObject:@""]; //Browser-Ext-Ver
+        [sequenceItems addObject:@""]; //BrowserCore-Ver
+    }
+    else
+    {
+        sequenceItems = [[sequence componentsSeparatedByString:kDelimeter] mutableCopy];
+    }
+    
+    // Validate count.
+    if (sequenceItems.count != kSequenceItemsCount)
```

Comment:
Should this be >= in case we increase count in the future?

Reply:
Good point, fixed.